src/sentry/static/sentry/app/views/settings/projectAlerts/issueEditor/index.tsx

```diff
@@ -103,13 +103,16 @@ class IssueRuleEditor extends AsyncView<Props, State> {
   }
 
   getEndpoints() {
-    const {
-      params: {ruleId, projectId, orgId},
-    } = this.props;
+    const {params, location} = this.props;
+    const {ruleId, projectId, orgId} = params;
+    const {__has_issue_alerts_targeting = 0} = location.query ?? {};
```

Comment:
Why the leading `__`?

Reply:
Idk, I can remove it. Just attempting to merge this on behalf of jeff

Reply:
Yeah this seems wrong since we pass around `issue_alerts_targeting` everywhere

```suggestion
    const {issue_alerts_targeting = 0} = location.query ?? {};
```

Reply:
Is this internal use only? I don't remember

Reply:
@billyvg Yeah, the idea is that before we migrate everyone we could ideally pass this to the querystring in the frontend, and then it'd forward it through to the backend. Then we can trial some rules using the new code.